webui/react/src/utils/experiment.test.ts

```diff
@@ -6,9 +6,11 @@ import * as utils from './experiment';
 describe('Experiment Utilities', () => {
   describe('isExperiment', () => {
     it('should validate experiment tasks', () => {
-      const experimentTask = {
+      const experimentTask: Type.ExperimentItem = {
```

Comment:
this change is not related to the error, but I saw type errors in this file. fixed it!